build-scripts/build_llvm.py

```diff
@@ -232,14 +244,17 @@ def main():
         "arc": {
             "repo": "https://github.com/llvm/llvm-project.git",
             "branch": "release/15.x",
+            "owner_project": "llvm/llvm-project",
```

Comment:
do we need to replicate that information? It's already available in the `repo` field.

Reply:
done